control/timeresp.py

```diff
@@ -212,38 +212,40 @@ def forced_response(sys, T=None, U=0., X0=0., transpose=False,
         Time steps at which the input is defined; values must be evenly spaced.
 
     U : array_like or float, optional
-        Input array giving input at each time `T` (default = 0).
+        Input array giving input at each time `T`
 
         If `U` is ``None`` or ``0``, a special algorithm is used. This special
         algorithm is faster than the general algorithm, which is used
         otherwise.
 
     X0 : array_like or float, optional
-        Initial condition (default = 0).
+        Initial condition.
 
     transpose : bool, optional
         If True, transpose all input and output arrays (for backward
-        compatibility with MATLAB and :func:`scipy.signal.lsim`).  Default
-        value is False.
+        compatibility with MATLAB and :func:`scipy.signal.lsim`).
 
-    interpolate : bool, optional (default=False)
+    interpolate : bool, optional
         If True and system is a discrete time system, the input will
         be interpolated between the given time steps and the output
         will be given at system sampling rate.  Otherwise, only return
         the output at the times given in `T`.  No effect on continuous
-        time simulations (default = False).
+        time simulations.
 
     return_x : bool, optional
-        If True (default), return the the state vector.  Set to False to
-        return only the time and output vectors.
+        - If False, return only the time and output vectors.
+        - If True, also return the the state vector.
+        - If None, determine the returned variables by
+          config.defaults['forced_response.return_x'], which was True
+          before version 0.9 and is False since then.
 
     squeeze : bool, optional
         By default, if a system is single-input, single-output (SISO) then
         the output response is returned as a 1D array (indexed by time).  If
         squeeze=True, remove single-dimensional entries from the shape of
         the output even if the system is not SISO. If squeeze=False, keep
         the output as a 2D array (indexed by the output number and time)
-        even if the system is SISO. The default value can be set using
+        even if the system is SISO. The default value can be overruled by
```

Comment:
overrided?

Reply:
I'll leave it to the native speaker. In any case it should be overruled vs overridden.

Reply:
I think you're right overridded is not a word. I vote overridden. Overruled suggests something that happens in a court case.